database/fixtures/recipes.js

```diff
@@ -0,0 +1,41 @@
+const Recipes = require("../models/recipes");
+
+const recipes = [
+  {
+    Title: "Spelt, Rye, and Whole Wheat Sourdough Bread",
+    Ingredients: {
+      "White flour": 520,
+      "White Spelt flour": 260,
+      "Whole-grain wheat flour": 208,
+      Water1: 781,
+      Water2: 104,
```

Comment:
Why do we need a Water2? What's the difference between adding Water2 + Water1 and having them separated?